### DISCUSSION TITLE
See RDS details on bref dashboard

### DISCUSSION THREAD

**Discussion by raiman264** [brefphp/bref#1181] [Support]
I just started using bref and downloaded the dashboard app
When I use it I can see logs, deployments, etc. But RDS databases is empty

I created the database by hand in the AWS console, what does it needs to have in order to be displayed in the Dashboard App?

**Comment by mnapoli** [comment]
Hi @raiman264, indeed it needs to be deployed in a CloudFormation stack to be visible. The Bref Dashboard works by letting you select an "app" in the home screen -> that's the CloudFormation stack.

If the database is not in a stack, then it will not be accessible through Bref Dashboard.
